src/Apps/Artist/ArtistApp.tsx

```diff
@@ -97,7 +99,7 @@ export const ArtistAppFragmentContainer = createFragmentContainer(ArtistApp, {
   `,
   me: graphql`
     fragment ArtistApp_me on Me {
-      ...RecentlyViewed_me
+      id
```

Comment:
Why even pluck `id`? I think we optimized this on the MP side to not fetch this from upstream (via `/api/v1/me` if its being provided in the header), but if the intent is to check if the user is logged in, why not just check `currentUser`?

Or, you can probably not even check here, and just make `RecentlyViewed` defensive to not being logged in (I think it already is).